docs/how-to-guides/rook-ceph-storage.md

```diff
@@ -0,0 +1,291 @@
+    operator = "Equal"
+    value    = "ceph"
+    effect   = "NoSchedule"
+  }
+
+  storage_class {
+    enable = true
+  }
+}
+```
+
+- `monitor_count` should be an odd number greater than three and not higher than the `count` variable of workers in the `worker_pool`.
+- `node_affinity` should match the `labels` variable provided in the `worker_pool`.
+- `toleration` should match the `taints` variable provided in the `worker_pool`.
+
+For information about all the available configuration options for the `rook-ceph` component, visit the component's [configuration reference](../configuration-reference/components/rook-ceph.md).
+
+#### Deploy the component
+
+Execute the following command to deploy the `rook-ceph` component:
+
+```bash
+lokoctl component apply rook-ceph
+```
+
+Verify the [OSD](https://docs.ceph.com/docs/master/glossary/#term-ceph-osd-daemon) pods in the `rook` namespace are in the `Running` state (this may take a few minutes):
+
+```console
+$ kubectl -n rook get pods -l app=rook-ceph-osd
+NAME                               READY   STATUS    RESTARTS   AGE
+rook-ceph-osd-0-6d4f69dbf9-26kzl   1/1     Running   0          15m
+rook-ceph-osd-1-86c9597b84-lmh94   1/1     Running   0          15m
+rook-ceph-osd-2-6d97697897-7bprl   1/1     Running   0          15m
+rook-ceph-osd-3-5bfb9d86b-rk6v4    1/1     Running   0          15m
+rook-ceph-osd-4-5b76cb9675-cxkdw   1/1     Running   0          15m
+rook-ceph-osd-5-8c86f5c6c-6qxtz    1/1     Running   0          15m
+rook-ceph-osd-6-5b9cc479b7-vjc9v   1/1     Running   0          15m
+rook-ceph-osd-7-7b84d6cc48-b46z9   1/1     Running   0          15m
+rook-ceph-osd-8-5868969f97-2bn9r   1/1     Running   0          15m
+```
+
+## Access the Ceph dashboard
+
+Ceph dashboard provides valuable visual information. It is an essential tool to monitor the Ceph cluster. Here are steps on how to access it.
+
+Obtain the password for the `admin` Ceph user by running the following command:
+
+```bash
+kubectl -n rook get secret rook-ceph-dashboard-password -o jsonpath="{['data']['password']}" | base64 --decode && echo
+```
+
+Execute the following command to forward port `8443` locally to the Ceph manager pod:
+
+```bash
+kubectl -n rook port-forward svc/rook-ceph-mgr-dashboard 8443
+```
+
+Now open the following URL: [https://localhost:8443](https://localhost:8443) and enter the username `admin` and the password obtained from the first step.
+
+## Enable and access toolbox
+
+Ceph is a complex software system, and not everything that happens in the Ceph cluster is visible at the `rook` layer of abstraction. So command-line interface to interact with Ceph cluster is useful to extract such hidden events and information. Ceph toolbox helps you access the ceph cluster using `ceph` CLI utility. Using the utility you can configure the Ceph cluster setting and debug the cluster.
+
+To deploy the toolbox, the `rook-ceph` component config should set the variable `enable_toolbox` to `true`.
```

Comment:
```suggestion
To deploy the toolbox, the `rook-ceph` component config should set the attribute `enable_toolbox` to `true`.
```

just to avoid confusion with hcl `variable` declaration.